challenge.js

```diff
@@ -58,4 +64,7 @@
 
 // ANSWER
 
+  /** node.appendChild()
+      https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild **/
```

Comment:
Right on. Since you mentioned `document.createElement` above, here is an example of using it in conjunction with this method you mention here to create a new element and attach it to the DOM:

Correct. All HTML elements are Nodes; if `div` is a reference to an on-DOM div, and we have a off-DOM element `e` we created in our JS, we can attach it with this method:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1